### ISSUE TITLE
Some calls to cvc4 fail

### ISSUE THREAD

**Issue by vblot** [smtcoq/smtcoq#39]
Some (but not all) calls to command `Lfsc_Checker` and tactic `smt` fail with:
```
System error: ".../smtcoq/examples/sat.plf: No such file or directory"
```
In file `examples/Example.v`, these calls are the ones on lines 50, 152, 164.

**Comment by ckeller** [comment]
Hi @vblot 
Did you do what is written in the section "Setting up environment for SMTCoq" at the end of [INSTALL.md](https://github.com/smtcoq/smtcoq/blob/master/INSTALL.md)?
If that solves the problem, I should probably put this section somewhere where it will be more visible.

**Comment by vblot** [comment]
Yes indeed I was missing the LFSCSIGS environment variable. Indeed I think it would be more visible if it were with the instructions for installing CVC4.

**Comment by ckeller** [comment]
Good idea: fixed by commit 7da3a3d
